monitoring/oplog-populator/dashboard.py

```diff
@@ -336,6 +348,7 @@ class Metrics:
                 total_number_buckets,
                 max_pipeline_size,
                 bandwidth,
+                retained_buckets,
```

Comment:
grafana layout does not divide by 7, so if we keep this extra panel we need to manually set some panel size to nicely fill the line.